pom.xml

```
@@ -76,16 +75,25 @@
         <maven.surefire.plugin.version>3.0.0-M7</maven.surefire.plugin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <sonar.jacoco.reportPath>../target/jacoco.exec</sonar.jacoco.reportPath>
-        <sonar.maven.version>3.3.0.603</sonar.maven.version>
-        <mockito.verson>4.11.0</mockito.verson>
-        <assertj.version>3.24.2</assertj.version>
+        <sonar.maven.version>3.9.1.2184</sonar.maven.version>
         <commons.io.version>2.11.0</commons.io.version>
-        <awaitility.version>4.2.0</awaitility.version>
         <checkstyle.excludes>**/jnosql/query/grammar/**</checkstyle.excludes>
         <weld.se.core.version>5.1.0.Final</weld.se.core.version>
         <yasson.version>3.0.2</yasson.version>
         <microprofile.config.version>3.0.2</microprofile.config.version>
-        <smallrye.config.version>3.1.3</smallrye.config.version>
+        <smallrye.config.version>3.2.1</smallrye.config.version>
+
+        <checkstyle.version>10.9.3</checkstyle.version>
+        <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
+        <nexus-staging-maven-plugin.version>1.6.13</nexus-staging-maven-plugin.version>
+
+        <!-- testing libraries -->
+        <junit.version>5.9.2</junit.version>
+        <mockito.verson>5.2.0</mockito.verson>
```

```suggestion
        <mockito.verson>4.11.0</mockito.verson>
```

There is a BK, we can fix on another PR.